http-add-on/templates/deployment-interceptor.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: interceptor
+    httpscaledobjects.http.keda.sh/version: {{ .Values.images.tag | default .Chart.AppVersion }}
+    keda.sh/addon: {{ .Chart.Name }}
+    app: {{ .Chart.Name }}
+    name: {{ .Chart.Name }}-interceptor
+    app.kubernetes.io/name: {{ .Chart.Name }}-interceptor
+    app.kubernetes.io/version: {{ .Values.images.tag | default .Chart.AppVersion }}
+    app.kubernetes.io/component: interceptor
+    app.kubernetes.io/part-of: {{ .Chart.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+{{ tpl .Values.additionalLabels . | indent 4}}
+  name: {{ .Chart.Name }}-interceptor
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: interceptor
+      httpscaledobjects.http.keda.sh/version: {{ .Values.images.tag | default .Chart.AppVersion }}
+      keda.sh/addon: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        control-plane: interceptor
+        httpscaledobjects.http.keda.sh/version: {{ .Values.images.tag | default .Chart.AppVersion }}
+        keda.sh/addon: {{ .Chart.Name }}
+        app: {{ .Chart.Name }}
+        name: {{ .Chart.Name }}-interceptor
+        app.kubernetes.io/name: {{ .Chart.Name }}-interceptor
+        app.kubernetes.io/version: {{ .Values.images.tag | default .Chart.AppVersion }}
+        app.kubernetes.io/component: interceptor
+        app.kubernetes.io/part-of: {{ .Chart.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    {{ tpl .Values.additionalLabels . | indent 4}}
+    spec:
+      containers:
+      - args:
+        image: "{{ .Values.images.interceptor }}:{{ .Values.images.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: '{{ .Values.interceptor.pullPolicy | default "Always" }}'
+        name: "{{ .Chart.Name }}-interceptor"
+        env:
+        # - name: KEDA_HTTP_OPERATOR_SERVICE_NAME
+        #   value: "{{ .Chart.Name }}-{{ .Values.operator.adminService }}"
+        # - name: KEDA_HTTP_OPERATOR_SERVICE_PORT
+        #   value: "{{ .Values.operator.adminPort }}"
+        # - name: KEDA_HTTP_OPERATOR_ROUTING_TABLE_PATH
+        #   value: "{{ .Values.operator.routingTableURLPath }}"
```

Comment:
Do we need them? If not, let's remove them

Reply:
removed in 0ac99c73b5d595d604f30d1fc665557f3e9f31b5